pkg/trimaran/loadvariationriskbalancing/analysis.go

```diff
@@ -83,7 +84,7 @@ func createResourceStats(metrics []watcher.Metric, node *v1.Node, podRequest *fr
 	// get resource usage statistics
 	nodeUtil, nodeStd, metricFound := getResourceData(metrics, watcherType)
 	if !metricFound {
-		klog.V(6).Infof("resource %s usage statistics for node %s: no valid data", watcherType, node.GetName())
+		klog.V(6).InfoS("Rsource usage statistics for node : no valid data", " node", klog.KObj(node))
```

Comment:
remove unneeded spaces, and fix typo:

```suggestion
		klog.V(6).InfoS("Resource usage statistics for node: no valid data", "node", klog.KObj(node))
```